src/Enzo/enzo_EnzoConfig.cpp

```diff
@@ -327,17 +339,47 @@ void EnzoConfig::pup (PUP::er &p)
   p | units_time;
 
   p  | method_grackle_use_grackle;
+
 #ifdef CONFIG_USE_GRACKLE
-  p  | method_grackle_use_cooling_timestep;
-  p  | method_grackle_radiation_redshift;
+  if (method_grackle_use_grackle) {
+    p  | method_grackle_use_cooling_timestep;
+    p  | method_grackle_radiation_redshift;
 
-  if (method_grackle_use_grackle){
-    if (p.isUnpacking()) { method_grackle_chemistry = new chemistry_data; }
-    p | *method_grackle_chemistry;
-  } else {
-    method_grackle_chemistry = NULL;
+    //    int is_null = (method_grackle_chemistry==NULL);
```

Comment:
Can remove this commented out code correct? (looks to me to bet the code from before the threadsafe Grackle PR